__tests__/get_all_pixels.spec.ts

```diff
@@ -0,0 +1,78 @@
+import request from 'supertest';
+import { BASE_URL, TOKEN } from '../helpers/constant'
+import { generateRandomString, generateRandomPixelType } from '../helpers/common'
+import { createPixel, deleteManyPixels } from '../helpers/api.helper'
+import { DEFAULT_TIMEOUT } from '../helpers/constant'
+
+describe('GET /api/pixels', function () {
+    var pixel_id: string[] = []
+
+    test('PIX-001 Verify that user can get list of total pixels', async function () {
+        const record_size = 2;
+        // create new pixels for testing
+        for (let i = 0; i < record_size; i++) {
```

Comment:
This step should be in beforeAll or beforeEach as pre-condition steps

Reply:
fixed